OpenRA.Mods.Common/Traits/Crates/Crate.cs

```diff
@@ -237,13 +231,12 @@ public bool CanEnterCell(CPos a, Actor ignoreActor = null, BlockedByActor check
 
 		bool ICrushable.CrushableBy(Actor self, Actor crusher, BitSet<CrushClass> crushClasses)
 		{
-			// Crate can only be crushed if it is not in the air.
-			return self.IsAtGroundLevel() && crushClasses.Contains(info.CrushClass);
+			return crushClasses.Contains(info.CrushClass);
 		}
 
 		LongBitSet<PlayerBitMask> ICrushable.CrushableBy(Actor self, BitSet<CrushClass> crushClasses)
 		{
-			return self.IsAtGroundLevel() && crushClasses.Contains(info.CrushClass) ? self.World.AllPlayersMask : self.World.NoPlayersMask;
+			return crushClasses.Contains(info.CrushClass) ? self.World.AllPlayersMask : self.World.NoPlayersMask;
```

Comment:
Logic in`Crate` for `INotifyCrushed.OnCrush` and `INotifyCrushed.OnCrush` also hardcodes crushability checks which check for airborne. This should be updated too. Perhaps ideally even made to reference the existing `CrushableBy` method to avoid duplication.

Reply:
My guess that they also also needed so the crate doesn't get crushed while still mid-air

Reply:
removed from `INotifyCrushed.OnCrush`